en/middleware/uorb.md

```diff
@@ -41,7 +41,7 @@ Then in the code, use them as topic id: `ORB_ID(offboard_mission)`.
 ## Publishing
 
 Publishing a topic can be done from anywhere in the system, including interrupt context (functions called by the `hrt_call` API).
-However, advertising a topic is only possible outside of interrupt context.
+However, the topic needs to be advertised and published once beforehand outside of interrupt context.
```

Comment:
```suggestion
However, the topic needs to be advertised and published outside of an interrupt context (at least once) before it can be published in an interrupt context.
```